src/reconciler.js

```diff
@@ -154,7 +154,7 @@ function applyEffect (fiber) {
   fiber.pending = fiber.pending || {}
   for (const k in fiber.effect) {
     const pend = fiber.pending[k]
-    pend && pend()

```

Comment:
This small change allows lazily writing lazy effects, e.g. `() => something()` rather than having to do `() => { something; }` to avoid returning a value.